cpp/include/cudf/detail/datetime.hpp

```diff
@@ -94,6 +94,36 @@ std::unique_ptr<cudf::column> extract_second(
   rmm::cuda_stream_view stream        = cudf::default_stream_value,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @copydoc cudf::extract_milli_second(cudf::column_view const&, rmm::mr::device_memory_resource *)
+ *
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ */
+std::unique_ptr<cudf::column> extract_milli_second(
+  cudf::column_view const& column,
+  rmm::cuda_stream_view stream        = cudf::default_stream_value,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
+/**
+ * @copydoc cudf::extract_micro_second(cudf::column_view const&, rmm::mr::device_memory_resource *)
+ *
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ */
+std::unique_ptr<cudf::column> extract_micro_second(
```

Comment:
I think `xyz_seconds` should be one word (rest of libcudf uses `xyzseconds`).
```suggestion
std::unique_ptr<cudf::column> extract_microsecond(
```

Reply:
Done 👍